src/setup/node.rs

```diff
@@ -67,11 +67,35 @@ impl Node {
         let config = NodeConfig::new();
         let meta = NodeMetaData::new()?;
 
-        Ok(Self {
+        let mut node = Self {
             config,
             meta,
             process: None,
-        })
+        };
+
+        // insert the config file cmd args
```

Comment:
I'm not too happy with the location of this code -- but my brain is too fried to think of a better place.

Happy to get suggestions.

Reply:
I'd probably make the change in `NodeMetaData::new` since that's where it's constructed from the args.

Reply:
We need access to `Node::config_filepath()` for the actual location of the generated file. We could move that down a layer as well?

Reply:
Works for me 👍